src/Installers/Windows/AspNetCoreModule-Setup/ANCMIISExpressV2/ancm_iis_expressv2.wxs

```diff
@@ -162,25 +162,43 @@
         <Directory Id="INSTALLLOCATION" ShortName="ANCM" Name="$(var.ProductName)">
           <Directory Id="VersionDir" Name="$(var.ProductVersionString)">
             <Component Id="AspNetCoreModule" Guid="84ed6ce6-c8a3-4fa8-a872-c98a1d15dd4f" Win64="$(var.IsWin64)">
-              <File Id="AspNetCoreModuleDll"
-                    Name="aspnetcorev2.dll"
-                    Source="$(var.AspNetCoreV2.TargetPath)"
-                    DiskId="1"
-                    Vital="yes">
-              </File>
+                <?if $(var.Platform) = "x64" ?>
+                    <File Id="AspNetCoreModuleDll"
+                        Name="aspnetcorev2.dll"
+                        Source="$(var.AspNetCoreV2.TargetPath)"
+                        DiskId="1"
+                        Vital="yes">
+                    </File>
+                <?else ?>
+                    <File Id="AspNetCoreModuleDll"
+                        Name="aspnetcorev2.dll"
+                        Source="$(var.AspNetCoreV2WoW64.TargetPath)"
```

Comment:
Why don't you just define a single variable in the .wixproj or pass it through using -d, then you don't need to duplicate the File elements.

Reply:
Right now these are defined via project references here: https://github.com/aspnet/AspNetCore/blob/master/src/Installers/Windows/AspNetCoreModule-Setup/Directory.Build.props#L45. In the wixproj file, can I just use $(AspNetCoreV2.TargetPath)?

Reply:
I tried a few things and nothing worked, ideas?

Reply:
if you have a .wixproj, you can do something like <DefineConstants>$(DefineConstants);Foo=$(SomePath)</DefineConstants>

Then in your .wixproj, you'll be able to access $(var.Foo). If you conditionally set $(SomePath) on the architecture you'll have one constant defined for both bitness

Reply:
Went with an approach where I define the constants in the wxs file instead (as the variables I wanted were defined in DefineConstants).